src/System.Runtime.Extensions/tests/System/IO/StreamWriterTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class StreamWriterTests
```

Comment:
>is there a way around that?

Yes, you'll need to add an explicit p2p reference like the ones removed in [this commit](https://github.com/dotnet/corefx/commit/963bbf421720f2d1a321f17360c2ad3c04d3d4da)